security/pod-to-pod-encryption.md

```diff
@@ -0,0 +1,60 @@
+
+### Value
+
+With no encryption enabled in a cluster setup, traffic between pods running on two different nodes is susceptible to many security issues (for example, snooping). Calico Enterprise comes with IPsec for encryption (which has been around for decades) to secure network traffic. {{site.prodname}} now offers WireGuard with state-of-the-art cryptography for both on-premises and cloud-hosted solutions.
+
+Calico sets up WireGuard tunnels between nodes providing transport-level security for pod-to-pod traffic. WireGuard provides [formally verified](https://www.wireguard.com/formal-verification/) secure and [performant tunnels](https://www.wireguard.com/performance/) without any specialized hardware. For a deep dive in to WireGuard implementation, see [whitepaper](https://www.wireguard.com/papers/wireguard.pdf).
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- Felix resource~/go/src/github.com/tigera/felix with WireGuard configuration parameters.
+
+### Before you begin...
+
+- Verify the operating system(s) running on the nodes in the cluster [support WireGuard](https://www.wireguard.com/install/).
+- Verify that [policies]({{ site.baseurl }}/reference/resources/hostendpoint) do not affect the WireGuard tunnel default port 51820 (UDP) on each node.
+
+   >**Note**: WireGuard in Calico does not support IPv6 at this time.
+   {: .alert .alert-info}
+
+### How to
+
+1. Install WireGuard on cluster nodes using these [instructions for your operating system](https://www.wireguard.com/install/).
+
+   >**Note**: Nodes that do not support WireGuard will not be secured by WireGuard tunnels, even if traffic running on the node to and from the pods goes to nodes that do support WireGuard.
+   {: .alert .alert-info}
+
+1. Enable WireGuard encryption across all the nodes using the following command.
+
+   ```
+   calicoctl patch felixconfiguration default --type='merge' -p '{"spec":{"wireguardEnabled":true}}'
+   ```
+
+For OpenShift, add the Felix configuration with WireGuard enabled [under custom resources]({{ site.baseurl }}/getting-started/openshift/installation#optionally-provide-additional-configuration).
+
+>**Note**: This above command can be used to change other WireGuard attributes.
+{: .alert .alert-info}
+
+To assign an IPv4 address to WireGuard interface created by Felix, you need to update the [Node resource]({{ site.baseurl }}/reference/resources/node).
+
+To disable WireGuard on a specific node with WireGuard installed, the host-specific Felix configuration can be adjusted, for example:
+```
+calicoctl patch felixconfiguration <Host-Name> --type='merge' -p '{"spec":{"wireguardEnabled":false}}'
+```
+
+For a list of other WireGuard parameters and configuration evaluation, see the [Felix resource]({{ site.baseurl }}/reference/resources/felixconfig#felix-configuration-definition).
+
+### Above and beyond
+
+- [Connections from Calico components to etcd]({{ site.baseurl }}/security/comms/crypto-auth)
```

Comment:
This link is fine for OS, but 3.0 Enterprise no longer supports etcd...maybe just link to "Secure Calico components"